src/main/java/spoon/support/reflect/reference/CtExecutableReferenceImpl.java

```diff
@@ -234,7 +234,8 @@ public boolean isConstructor() {
 	@Override
 	public boolean isOverriding(CtExecutableReference<?> executable) {
 		CtExecutable<?> exec = executable.getDeclaration();
-		if (exec == null) {
+		CtExecutable<?> thisExec = getDeclaration();
```

Comment:
why? Because I used same approach like old code. 
Anyway you are right. So I have replaced that on 3 places. Now it uses better `getExecutableDeclaration`